tests/unittests/GraphOptzTest.cpp

```diff
@@ -357,7 +357,7 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvWithReshapeConst) {
 /// Check that the batch normalization optimization is
 /// not blocked by predicates and that it preserves them.
 TEST_F(GraphOptz, optimizeBatchNormAfterConvWithPred) {
-  Node *A =
+  Placeholder *A =
```

Comment:
You use Placeholder here and Node for the next line, which is inconsistent. Either use `Placeholder *` for both or `auto *` for both.